Kalk88/GrowBud
# Add fetch for schedules and render list

## **Description**

A view for myWateringSchedules has been created. Very bare minimum functionality.

## **How to test?**

Build project and Login
Add a schedule from the client and check that the name is show in the My Schedules list
